config/environments/development.rb

```diff
@@ -30,13 +30,6 @@
   # Expands the lines which load the assets
   config.assets.debug = true
 
-  # Raise exception on mass assignment protection for Active Record models
-  config.active_record.mass_assignment_sanitizer = :strict
-
-  # Log the query plan for queries taking more than this (works
-  # with SQLite, MySQL, and PostgreSQL)
-  config.active_record.auto_explain_threshold_in_seconds = 0.5
```

Comment:
For the record: https://github.com/rails/rails/commit/d3688e02ca52c0b72d3092e8498da51e06b7fc58